app/assets/stylesheets/application.scss

```diff
@@ -32,3 +32,56 @@
   position: static;
   width: 100%;
 }
+
+.right_text{
```

Comment:
Selector `right_text` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space